Added draw test hand functionallity

Niebvelungen opened the pull request.

Kossetsoup commented on cardDatabase/templates/cardDatabase/html/view_decklist.html:
```diff
@@ -159,6 +187,15 @@ <h4>Card cost distribution</h4>
                     mobile
                 );
 
+                let mainDeckCards = buildCardWrappers(cards);
```
I think that since we refer to specific zone names elsewhere it might be worth just having this access "Main", "Main Deck", etc. as usual. I say this because this will pull from any "maybeboard" zones that people add for tech cards they're considering so it may mess up their drawing from their deck.

Kossetsoup reviewed (changes requested):
Just one issue to do with where it pulls the cards from to do the test hands, I think it should only refer to the main deck. For now we can do this by name, but in the future it should refer to whatever zone is tagged as the main deck.

Niebvelungen commented:
@Kossetsoup i reworked the feature to be more visually appealing.

Its now under the stats

I also added a multi select to change zones from stats and test hand from.

Sadly i can use mdb because it implements the stats graphs to some decree and if i add the lib it destroys the graphs.

![2024-01-24_22h09_04](https://github.com/Force-of-Wind/fowsim/assets/74566937/c0da51df-6337-424e-9263-0ea40c9e3665)
![2024-01-24_22h09_09](https://github.com/Force-of-Wind/fowsim/assets/74566937/2ac3d7a7-8e32-4ae9-8443-58a4e39f8241)
![2024-01-24_22h09_43](https://github.com/Force-of-Wind/fowsim/assets/74566937/b0915c9a-aadd-4304-8edc-5766cbec72cb)

Kossetsoup commented:
Looks too, I like it! Maybe just one more thing: add a title above the draw test hand section similar to what you have for "Card cost distribution" and "Attribute distribution in %"
